packages/core/src/amazonq/webview/ui/connector.ts

```diff
@@ -130,8 +135,8 @@ export class Connector {
 
     transform = (tabID: string): void => {
         switch (this.tabsStorage.getTab(tabID)?.type) {
-            default:
-                this.cwChatConnector.transform(tabID)
+            case 'gumby':
+                this.gumbyChatConnector.transform(tabID)
                 break
         }
```

Comment:
Is this a switch with one entry? Could we make it into an if?

Reply:
I think this is a good suggestion, it may better convey that `/transform` isn't an action we ever expect another app to handle